apps/epic-react/src/pages/discord/redirect.tsx

```diff
@@ -0,0 +1,12 @@
+export async function getServerSideProps() {
+  return {
+    redirect: {
+      destination: process.env.NEXT_PUBLIC_DISCORD_INVITE_URL,
+      permanent: true,
+    },
+  }
+}
```

Comment:
this pitches them right into discord otherwise it's all in the background and people get confused